daphne/src/roles/leader.rs

```diff
@@ -346,9 +346,16 @@ pub trait DapLeader<S>: DapAuthorizedSender<S> + DapAggregator<S> {
             )
             .await?;
         let (state, agg_job_init_req) = match transition {
-            DapLeaderTransition::Continue(state, agg_job_init_req) => (state, agg_job_init_req),
-            DapLeaderTransition::Skip => return Ok(0),
-            DapLeaderTransition::Uncommitted(..) => {
+            DapLeaderAggregationJobTransition::Continued(state, agg_job_init_req) => {
+                (state, agg_job_init_req)
+            }
+            DapLeaderAggregationJobTransition::Finished(agg_span)
+                if agg_span.report_count() == 0 =>
```

Comment:
I removed `Skip` because it feels a bit redundant. But the fact that we need this `if` statement here makes me wonder if we really want an empty agg span to distinguish between these branches.

Reply:
why do we need this if?

Reply:
If the agg span is empty, then we quite the agg job. Otherwise, if the agg span is non-empty, we send the next request for the agg job.